src/kubernetes-ingress-controller/guides/using-kongplugin-resource.md

```diff
@@ -37,17 +42,24 @@ This is expected as Kong does not yet know how to proxy the request.
 
 ## Installing sample services
 
-We will start by installing two services,
-an echo service and an httpbin service.
+We will start by installing two services. First an httpbin service:
+
+```sh
+kubectl apply -f https://bit.ly/k8s-httpbin
+```
 
-```bash
-$ kubectl apply -f https://bit.ly/k8s-httpbin
+```sh
```

Comment:
Can you preface each output block with "Expected output:"?

Reply:
I did that at first, and thought it was unnecessarily redundant.  If you feel strongly I can add it back in

Reply:
Hmm. I see what you mean. Let's go with your way for now, and we can always revisit if people continue to run into issues.